docs/Concepts/TLS.md

```diff
@@ -0,0 +1,18 @@
+---
+description: Secure communication using TLS
+---
+
+# TLS Communication
+
+EthSigner supports TLS to secure inbound and outbound HTTP JSON-RPC
+requests, and [communication with Hashicorp Vault](../HowTo/Store-Keys/Use-Hashicorp.md).
+
+![EthSigner TLS](../images/Ethsigner_TLS.png)
+
+
+Private keys and certificates for client and server TLS connections must be
+stored in password-protected PKCS #12 keystores. The server 
+(in this example, Hyperledger Besu) must be configured to accept TLS
+connections.
+
+Use the command line options to [enable and configure](../HowTo/Configure-TLS.md) TLS.
```

Comment:
This sentence only links to the content on enabling and configuring TLS on the HTTP JSON-RPC connection.  Is it worth also linking the the configuring Hashicorp section as well so it's clear there's two places that need configured?

Reply:
Done.